locksmith/__tests__/controllers/metadataController.test.ts

```diff
@@ -42,4 +79,32 @@ describe('Metadata Controller', () => {
       )
     })
   })
+
+  describe('updateDefaults', () => {
```

Comment:
Can we add tests to make sure that the signature is checked and that updating fails if the signature does not match?